packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/tiles_inspector.dart

```diff
@@ -208,269 +207,574 @@ void runTests() {
     );
   }, skip: isWeb /* Tiles not supported on the web */);
 
-  /// Check that two lists of [WeightedLatLng] are more or less equal.
-  void expectHeatmapDataMoreOrLessEquals(
-    List<WeightedLatLng> data1,
-    List<WeightedLatLng> data2,
-  ) {
-    expect(data1.length, data2.length);
-    for (int i = 0; i < data1.length; i++) {
-      final WeightedLatLng wll1 = data1[i];
-      final WeightedLatLng wll2 = data2[i];
-      expect(wll1.weight, wll2.weight);
-      expect(wll1.point.latitude, moreOrLessEquals(wll2.point.latitude));
-      expect(wll1.point.longitude, moreOrLessEquals(wll2.point.longitude));
+  group('Heatmaps', () {
```

Comment:
Added Heatmap tests to own grounp, like tile overlay tests and ground overlay tests.